pkg/apis/sources/v1/container_lifecycle_test.go

```diff
@@ -104,24 +104,36 @@ func TestContainerSourceStatusIsReady(t *testing.T) {
 		}(),
 		wantConditionStatus: corev1.ConditionUnknown,
 		want:                false,
+	}, {
+		name: "mark ready sa",
+		s: func() *ContainerSourceStatus {
+			s := &ContainerSourceStatus{}
+			s.InitializeConditions()
+			s.MarkOIDCIdentityCreatedSucceeded()
+			return s
+		}(),
+		wantConditionStatus: corev1.ConditionUnknown,
+		want:                false,
 	}, {
 		name: "mark ready sb and ra",
 		s: func() *ContainerSourceStatus {
 			s := &ContainerSourceStatus{}
 			s.InitializeConditions()
 			s.PropagateSinkBindingStatus(&readySinkBinding.Status)
 			s.PropagateReceiveAdapterStatus(availableDeployment)
+			s.MarkOIDCIdentityCreatedSucceeded()
 			return s
 		}(),
 		wantConditionStatus: corev1.ConditionTrue,
 		want:                true,
 	}, {
-		name: "mark ready sb and unavailable ra",
+		name: "mark ready sb and unavailable ra ",
 		s: func() *ContainerSourceStatus {
 			s := &ContainerSourceStatus{}
 			s.InitializeConditions()
 			s.PropagateSinkBindingStatus(&readySinkBinding.Status)
 			s.PropagateReceiveAdapterStatus(unavailableDeployment)
+			s.MarkOIDCIdentityCreatedFailed("unknown", "")
```

Comment:
Here we should use `s.MarkOIDCIdentityCreatedSucceeded()`, as we want to check false condition of the deployment status. Otherwise we could get a falsepositive.